airbyte-webapp/src/packages/cloud/views/layout/CloudMainView/CloudMainView.module.scss

```diff
@@ -1,10 +1,47 @@
-@use "../../../../../scss/colors";
-@use "../../../../../scss/variables";
-@forward "../../../../../views/layout/SideBar/SideBar.module.scss";
+@use "scss/colors";
+@use "scss/variables";
 
 $sidebar-workspace-font-size: 9px;
 $sidebar-workspace-font-weight: 400;
 
+$banner-height: 30px;
+
+.mainContainer {
+  overflow: hidden;
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  height: 100%;
+  min-height: 680px;
+
+  .content {
+    overflow-y: auto;
+    width: 100%;
+    height: 100%;
+
+    .dataBlock {
+      width: 100%;
+      height: 100%;
+    }
+
+    &.alertBanner {
+      margin-top: $banner-height;
+
+      .dataBlock {
+        height: calc(100% - #{$banner-height});
+      }
+    }
+
+    &.speedyConnectionBanner {
+      margin-top: 50px;
+
+      .dataBlock {
+        height: calc(100% - 50px);
+      }
+    }
+  }
+}
+
 .workspaceButton {
```

Comment:
I think we'll need a width definition here to avoid overflowing:

![image](https://user-images.githubusercontent.com/7550957/217283780-0942cd74-a286-4fa7-9f64-3d40ccb9f1cd.png)